docs/api/filter-gateway.md

```diff
@@ -0,0 +1,1575 @@
+# Vald Filter Gateway APIs
+
+## Overview
+
+Filter Servie is responsible for providing insert, update, upsert and search interface for `Vald Filter Gateway`.
+
+```rpc
+service Filter {
+    // insert, update, upsert and search RPCs
+}
+```
+
+## Insert RPC
+
+Insert RPC is the method to insert object through Vald Filter Gateway.
+
+```rpc
+  rpc InsertObject(payload.v1.Insert.ObjectRequest)
+      returns (payload.v1.Object.Location) {
+    option (google.api.http) = {
+      post : "/insert/object"
+      body : "*"
+    };
+  }
+  ```
```

Comment:
```suggestion
```rpc
service Filter {
  rpc InsertObject(payload.v1.Insert.ObjectRequest)
      returns (payload.v1.Object.Location) {
    option (google.api.http) = {
      post : "/insert/object"
      body : "*"
    };
  }
```
```

Please format as same as other API docs.
And, please fix others in this doc.

Reply:
fixed :)